CK2Plus_expanded/common/dynasties/00_dynasties.txt

```diff
@@ -44679,12 +44679,6 @@
 	religion = baltic_pagan

```

Comment:
The changes in here can all be dropped because afaik ?! you already tended to these problems.